app/parser/parser_utils.py

```diff
@@ -1,42 +1,134 @@
+# -*- coding: utf-8 -*-
+'''Parser Utils Module
+
+This module defines a ParserUtils class with several static methods to be used
+by parser implementations.
+
+'''
+
 from .schema_parser_exception import SchemaParserException
 
 
 class ParserUtils(object):
-    # gets the value associated with key in the dict, or throws an exception if it's not present
+    '''ParserUtils
+
+    The ParserUtils class exposes a number of static utility methods that are
+    intended to be called from within parser implementations.
+
+    '''
+
     @staticmethod
     def get_required(obj, key):
+        '''Get a required value from the dict.
+
+        Gets the value associated with key in the dict, or raises a
+        SchemaParserException if it's not present.
+
+        Args:
```

Comment:
should be 
:param obj: A dict on json object
:param key: The name of the property to retrieve
:returns: the value if found